CHANGES.rst

```diff
@@ -2,6 +2,13 @@
 Changes
 =======
 
+v1.4 (2022-04-07)
```

Comment:
So I guess that means you want me to release v1.4 today ...

I usually just write `development version` as the title, with neither date nor version number, and then update it just before making the release.

(But no problem, I can make a release today.)